webapp/sources/rudder/rudder-core/src/test/resources/database.properties

```diff
@@ -1,4 +1,4 @@
 jdbc.driverClassName = org.postgresql.Driver
-jdbc.url = jdbc:postgresql://localhost:5432/test
-jdbc.username = test
-jdbc.password = test
+jdbc.url = jdbc:postgresql://localhost:55432/test
```

Comment:
Isn't that specific to your own environement ?

Reply:
totally!